styles/components/pagination/_pagination.scss

```diff
@@ -1,29 +1,39 @@
-@import "../../global/index";
+@import '../../global/index';
 
 .usa-dt-pagination {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    flex-wrap: wrap;
-    .usa-dt-pagination__totals {
-        flex: 1 1 auto;
-        text-align: left;
-        font-size: $base-font-size;
-    }
-    .usa-dt-pagination__wrapper {
-        display: flex;
-        justify-content: flex-end;
-        align-items: center;
-        flex-wrap: wrap;
-        &:first-child {
-            // if there is no description / results text
-            justify-content: flex-start;
-        }
-        @media(max-width: $small-screen) {
-            justify-content: flex-start;
-        }
-        @import './pager';
-        @import './limitSelector';
-        @import './goTo';
-    }
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	flex-wrap: wrap;
+
+	@media (max-width: $medium-screen - 1) {
```

Comment:
can you replace this with the mixins? https://github.com/fedspendingtransparency/data-transparency-ui/blob/785dc47feceeb4400397bce57ae653d4fb974e5b/styles/global/_mixins.scss

Reply:
thanks for pointing those out; done @MaxwellKendall